trino/sqlalchemy/dialect.py

```diff
@@ -231,7 +231,7 @@ def get_view_definition(self, connection: Connection, view_name: str, schema: st
         """
         ).strip()
         res = connection.execute(sql.text(query), schema=schema, view=view_name)
-        return res.scalar()
+        return res.scalar_one_or_none()
```

Comment:
can the sqlalchemy changes be a preparatory commit? Some context in the commit message will be useful (either as separate commit if possible or in current commit if not).

Reply:
Done